### PR TITLE
Added Form Validation

### PR DESCRIPTION
**Pull request by jamal-pb95**

### CODE REVIEW

File: src/pages/javascript/form-validation/index.md [hunk 1]
```diff
@@ -2,14 +2,147 @@
...
+            <tr>
+               <td align="right"></td>
+               <td><input type="submit" value="Submit" /></td>
+            </tr>
+            
+         </table>
+      </form>
+      
+   </body>
+</html>'
```
Comment (IsaacAbrahamson) [inline src/pages/javascript/form-validation/index.md]: Again, three backticks after not one.
'''
```

File: src/pages/javascript/form-validation/index.md [hunk 2]
```diff
@@ -2,14 +2,147 @@
...
+# Basic Validation - First of all, the form must be checked to make sure all the mandatory fields are filled in. It would require just a loop through each field in the form and check for data.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+# Data Format Validation − Secondly, the data that is entered must be checked for correct form and value. Your code must include appropriate logic to test correctness of data.
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+## Example:
 
+'<html>
```
Comment (IsaacAbrahamson) [inline src/pages/javascript/form-validation/index.md]: Please make sure you are using the correct code markdown. You should have three backticks (not one) above and below each code segment, like this (I'm using a single quote for example):
```
'''
code
'''
```
Please preview your changes to see the problem.

File: src/pages/javascript/form-validation/index.md [hunk 3]
```diff
@@ -2,14 +2,147 @@
...
+   </body>
+</html>'
 
+# Output : lookout [here](https://liveweave.com/LP9eOP)
+
+## Basic Form Validation
+
+First let us see how to do a basic form validation. In the above form, we are calling validate() to validate data when onsubmit event is occurring. The following code shows the implementation of this validate() function.
+
+'<script type="text/javascript">
```
Comment (IsaacAbrahamson) [inline src/pages/javascript/form-validation/index.md]: Check backticks

File: src/pages/javascript/form-validation/index.md [hunk 4]
```diff
@@ -2,14 +2,147 @@
...
+         
+         if( document.myForm.Country.value == "-1" )
+         {
+            alert( "Please provide your country!" );
+            return false;
+         }
+         return( true );
+      }
+   //-->
+</script>'
```
Comment (IsaacAbrahamson) [inline src/pages/javascript/form-validation/index.md]: Check backticks

File: src/pages/javascript/form-validation/index.md [hunk 5]
```diff
@@ -2,14 +2,147 @@
...
+
+## Data Format Validation
+
+Now we will see how we can validate our entered form data before submitting it to the web server.
+
+The following example shows how to validate an entered email address. An email address must contain at least a ‘@’ sign and a dot (.). Also, the ‘@’ must not be the first character of the email address, and the last dot must at least be one character after the ‘@’ sign.
+
+# Example:
+
+'<script type="text/javascript">
```
Comment (IsaacAbrahamson) [inline src/pages/javascript/form-validation/index.md]: Check backticks

File: src/pages/javascript/form-validation/index.md [hunk 6]
```diff
@@ -2,14 +2,147 @@
...
+         if (atpos < 1 || ( dotpos - atpos < 2 )) 
+         {
+            alert("Please enter correct email ID")
+            document.myForm.EMail.focus() ;
+            return false;
+         }
+         return( true );
+      }
+   //-->
+</script>'
```
Comment (IsaacAbrahamson) [inline src/pages/javascript/form-validation/index.md]: Check backticks

**Comment by Bouncey** [comment]
Thanks for the article @jamal-pb95 :+1:

**Pull request merged**